rflx/model/message.py

```diff
@@ -2023,7 +2035,17 @@ def _merge_inner_message(
             },
         }
 
-        structure, types = self._prune_dangling_fields(structure, types)
+        byte_order = {
+            **{f: b for f, b in message.byte_order.items() if f != field},
+            **{
+                f: b
+                for f, b in inner_message.byte_order.items()
+                if inner_message.identifier not in message_arguments
+                or f.identifier not in message_arguments[inner_message.identifier]
+            },
+        }
```

Comment:
My aim was to have `types` and `byte_order` to have the same keys, so we need the same condition here as above, right?

Reply:
I don't think it makes sense to have the same keys for `types` and `byte_order` in all cases. In case of parameterized messages, `types` also contains entries for the parameters of a message. I see no use for defining the byte order for message parameters.

I guess the condition is only relevant for `types` in case of parameterized messages, but doesn't make a difference for `byte_order`.

Reply:
Makes sense, I will remove this condition.

Reply:
done